rfc/rfc004-package-log.md

```diff
@@ -12,8 +12,21 @@ http://peter.bourgon.org/go-kit/#package-log
 
 ## Scope
 
-To be defined.
+* Log SHALL be drop in compatible with the standard library logger.
+* Log SHALL use key / value pairs for structured data.
+* Log MAY allow multiple stored logging contexts each with a different set of k/v pairs.
+* Log SHALL be transport agnostic by accepting and io.Writer as output target.
+* Log SHALL provide a default stdout io.Writer.
+* Log MAY implement io.MultiWriters allowing broadcast of logs over multiple transports.
```

Comment:
Would it be worth it to use channels instead of writers, and just have log transports give a channel to the log package, and have the log package multiplex out to all the log transport channels?  Then the transport could decide buffer on the channel and such.  Maybe if a channel would block the send, skip that channel?